data/human/culture conversations.txt

```diff
@@ -971,6 +971,99 @@ mission "Exotic Life: Glaze"
 			`	You walk back to your ship from the binoculars.`
 				decline
 
+mission "Exotic Life - Bluestone"
+	minor
+	source Bluestone
+	to offer
+		random < 10
+	on offer
+		conversation
+			`You step off your ship and wander around Bluestone's quiet yet functional spaceport. The buildings, made mostly of durable plastic resin, form a quaint but near-uniform red-brown landscape, except for the starkly contrasting cheerful small yellow pub in the center. The spaceport is quiet, but the low hum of activity from a nearby oil drill gives it some life. As you are entering the pub for a drink, you spot an announcement on a bulletin board:`
+			`	"Guided Tour: Hybrid Crop Research Village - an exceptional hub of field-based research led by the Cornucopia College of Agriculture, nestled within captivating landscapes!"`
+			`	The bulletin catches your eye. It's a rare chance to see something other than refinery equipment and plastics, and the mention of hybrid crops entails something different. There's a signup sheet, and only a few spots remain.`
+			choice
+				`	(Sign up.)`
+				`	(Skip the tour.)`
+					decline
+			`	After signing up for the next tour, you are directed to hop onto a beat-up old bus that rattles along to the research village. You notice Bluestone's red-brown industrial sprawl transition into the rocky fields of deep, desaturated blue grasses that the planet is known for. The village itself appears a small yet dynamic community, centered around a sleek, glass-paneled laboratory and greenhouse emanating reddish-yellow light. Dotted around are modest plastic resin apartment buildings, with the entire community surrounded by an expansive field covering rows and rows of crops.`
+			`	You get off the bus and are immediately ushered into the laboratory by a young-looking tour guide, along with half a dozen other tourists. She has a small nametag that reads:`
+			`		Zea S.`
+			`		Community Tour Guide`
+			`		MS in Agronomy, CCA`
+			`		(Cornucopia College of Agriculture)`
```

Comment:
It seems weird to have a name tag that explains the abbreviation for a college. Maybe:
```suggestion
			`		MS in Agronomy`
			`		Cornucopia College of Agriculture`
```